subsys/bluetooth/controller/Kconfig

```diff
@@ -750,4 +750,18 @@ config BT_MAYFLY_YIELD_AFTER_CALL
 	  If set to 'n', all pending mayflies for callee are executed before
 	  yielding
 
+config BT_CTLR_USER_EXT
+	prompt "Enable proprietary extensions in Controller"
+	bool
+	help
+	  Catch-all for enabling proprietary event types in Controller behavior.
+
+config BT_CTLR_USER_EXT_RANGE
+	int "Range of contants reserved for proprietary event types"
+	default 5
+	range 0 10
+	help
+	  Range of contants reserved for proprietary event types. The range
```

Comment:
con**s**tants. 

nit: Perhaps "constants" don't convey the meaning well enough; what about "Number of event types to reserve for proprietary use"

Reply:
Changed working as suggested